build/rust/Cargo.lock

```diff
@@ -330,6 +345,12 @@ dependencies = [
  "serde",
 ]
 
+[[package]]
+name = "cc"
```

Comment:
what is this needed for? We should not be building cpp files from cargo so this is a banned feature